frontend/src/pages/AccountManagement/index.js

```diff
@@ -303,6 +360,13 @@ function AccountManagement({ updateUser }) {
     minute: '2-digit',
   }).format(new Date(user.lastLogin));
 
+  const userHasEmailRoles = () => {
+    const userRoles = user.roles.map((role) => role.name);
+    return userRoles.some((role) => submitsForApprovalRoles.includes(role))
+      || userRoles.some((role) => managerAndCollaboratorRoles.includes(role))
+      || userRoles.some((role) => recipientsAvailable.includes(role));
```

Comment:
```suggestion
    return userRoles.some((role) => [
      ...submitsForApprovalRoles,
      ...managerAndCollaboratorRoles,
      ...recipientsAvailable].includes(role)));
```

minor, just one lookup instead of three

Reply:
Updated.